release/src/Types.hs

```diff
@@ -62,10 +68,24 @@ type TextVersion = Text
 type GroupId = [Text]
 type ArtifactId = Text
 type Classifier = Text
+type ArtifactType = Text
+
+-- Fully qualified coordinates for a Maven artifact.
+data MavenCoords = MavenCoords {
```

Comment:
The formatting is a bit off for Haskell. The normal formatting is:

```Haskell
data MavenCoords = MavenCoords
    { groupId :: !GroupId
    , artifactId :: !ArtifactId
    , version :: !TextVersion
    , classifier :: !(Maybe ArtifactType)
    , artifactType :: !ArtifactType
    } deriving Show
```

Reply:
implemented In 2d9b593a6